app/controllers/councillor_contributions_controller.rb

```diff
@@ -69,6 +69,16 @@ def show
     end
   end
 
+  def mark_as_reviewed
+    @councillor_contribution = CouncillorContribution.find(params[:id])
+    if @councillor_contribution.reviewed?
```

Comment:
This logic can be replaced by [the handy `toggle` method](http://api.rubyonrails.org/v4.2.7.1/classes/ActiveRecord/Persistence.html#method-i-toggle).

```
@councillor_contribution.toggle('reviewed')
@councillor_contribution.save!
```